<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Table</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1262</width>
      <height>681</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QPushButton" name="updateButton">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>30</y>
       <width>170</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Обновить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addElement">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>610</y>
       <width>340</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Добавить элемент</string>
     </property>
    </widget>
    <widget class="QPushButton" name="editElement">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>610</y>
       <width>340</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Изменить элемент</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteElemet">
     <property name="geometry">
      <rect>
       <x>840</x>
       <y>610</y>
       <width>340</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Удалить элемент</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>240</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Список подарков:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="xlxsButton">
     <property name="geometry">
      <rect>
       <x>1060</x>
       <y>530</y>
       <width>120</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Экспорт в xlxs</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="table">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>1080</width>
       <height>460</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEmail">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>530</y>
       <width>260</width>
       <height>30</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="editElementReady">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>530</y>
       <width>170</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Изменить готовность</string>
     </property>
    </widget>
    <widget class="QLabel" name="pictureQ">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1291</width>
       <height>681</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../photos/listback.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <zorder>pictureQ</zorder>
    <zorder>updateButton</zorder>
    <zorder>addElement</zorder>
    <zorder>editElement</zorder>
    <zorder>deleteElemet</zorder>
    <zorder>label</zorder>
    <zorder>xlxsButton</zorder>
    <zorder>table</zorder>
    <zorder>textEmail</zorder>
    <zorder>editElementReady</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuexit">
    <property name="title">
     <string>Exit</string>
    </property>
    <addaction name="actionOpen_login_window"/>
   </widget>
   <addaction name="menuexit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_login_window">
   <property name="text">
    <string>Open login window</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../photos/background_q.qrc"/>
 </resources>
 <connections/>
</ui>
